src/backend/distributed/executor/executor_util_tuples.c

```diff
@@ -0,0 +1,129 @@
+/*-------------------------------------------------------------------------
+ *
+ * executor_util_tuples.c
```

Comment:
very minor: we are already under `executor` folder, so might omit `executor_` in the file name   (as we don't have a specific header for this file but use executor_util.h)